src/sidebar/services/streamer.ts

```diff
@@ -85,6 +90,11 @@ export class StreamerService {
     const updates = Object.values(this._store.pendingUpdates());
     if (updates.length) {
       this._store.addAnnotations(updates);
+      // Highlight the new/edited annotations for 5 seconds
+      this._store.highlightAnnotations(
+        updates.map(({ id }) => id).filter(Boolean) as string[],
+      );
+      this._window.setTimeout(() => this._store.highlightAnnotations([]), 5000);
```

Comment:
We might end up removing this from here or changing this logic. This is just a temporary simplistic approach so that annotations are not highlighted forever.

Reply:
I definitely think a simple 5 second timeout is not going to be workable in a class activity where there are tens or hundreds of annotations. It's OK for a first PR though to start to show what the highlight effect will look like.